src/opera/test/data/test_dswx_s1_config.yaml

```diff
@@ -25,8 +28,9 @@ RunConfig:
                 ProgramOptions:
                     - 'hello world > dswx_s1_pge_test/output_dir/dswx_s1_test.txt;'
                     - '/bin/echo DSWx-S1 invoked with RunConfig'
-                ErrorCodeBase: 200000
+                ErrorCodeBase: 100000
```

Comment:
Not critical, but this is the error code base used by DSWx-HLS. The idea was to have each PGE use a unique base value, and I've incremented by `100000` for each new PGE, so the "official" offset for DSWx-S1 should be `400000`